components/expression_language/syntax.rst

```diff
@@ -139,6 +139,23 @@ constant::
 
 This will print out ``root``.
 
+The second one is ``enum()``, which will return the case of an enumeration::
+
+    enum Foo
+    {
+        case Bar;
+    }
+
+    var_dump(App\Enum\Foo::Bar === $expressionLanguage->evaluate(
+        'enum("App\\\\Enum\\\\Foo::Bar")'
```

Comment:
I used your code in my project and only needed 3 backslashes when defining the expression via attribute

Reply:
You're right, updated!

Reply:
Thanks